styled-switch-view/deploy.gradle

```diff
@@ -0,0 +1,23 @@
+apply plugin: 'com.novoda.bintray-release'
+
+publish {
+    bintrayUser = System.getenv('BINTRAY_USER')
+    bintrayKey = System.getenv('BINTRAY_API_KEY')
+
+    repoName = 'android'
+    uploadName = 'styled-switch-view'
```

Comment:
🤟Thanks for taking care of this as well. Do you want to set up the Bintray or me?